src/VisualStudio/Core/Test/PullMemberUp/PullMemberUpViewModelTest.vb

```
@@ -0,0 +1,336 @@
+
+            'Dependents of Goo
+            Assert.True(FindMemberByName("Goo()", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("e", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("What", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("BarBar(double)", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("Nested1()", viewModel.Members).IsChecked)
+            Assert.True(FindMemberByName("gravitational", viewModel.Members).IsChecked)
+
+            'Not the depenents of Goo
+            Assert.False(FindMemberByName("i", viewModel.Members).IsChecked)
+            Assert.False(FindMemberByName("FooEvent", viewModel.Members).IsChecked)
+        End Function
+
+        Private Function FindMemberByName(name As String, memberArray As ImmutableArray(Of PullMemberUpSymbolViewModel)) As PullMemberUpSymbolViewModel
+            Dim member = memberArray.FirstOrDefault(Function(memberViewModel) memberViewModel.MemberName.Equals(name))
+            If (member Is Nothing) Then
+                Assert.True(False, $"No member called {name} found")
+            End If
+            Return member
+        End Function
+
+        Private Async Function GetViewModelAsync(markup As XElement, languageName As String) As Task(Of PullMemberUpDialogViewModel)
+            Dim workspaceXml =
+            <Workspace>
+                <Project Language=<%= languageName %> CommonReferences="true">
+                    <Document><%= markup.NormalizedValue.Replace(vbCrLf, vbLf) %></Document>
+                </Project>
+            </Workspace>
+
+            Using workspace = TestWorkspace.Create(workspaceXml)
+                Dim doc = workspace.Documents.Single()
+                Dim workspaceDoc = workspace.CurrentSolution.GetDocument(doc.Id)
+                If (Not doc.CursorPosition.HasValue) Then
+                    Assert.True(False, "Missing caret location in document.")
```

You can just throw an exception with the message.